src/containers/AboutIndex/components/WorkDoneCard.tsx

```diff
@@ -0,0 +1,44 @@
+import { makeStyles, Typography } from '@material-ui/core';
+import Paper from 'components/layout/Paper';
+import React from 'react';
+import MarkdownRenderer from 'components/miscellaneous/MarkdownRenderer';
+import BuildIcon from '@material-ui/icons/Build';
+
+export type WorkDoneCardProps = {
+  changelogURL: string;
+  title: string;
+};
+
+const useStyles = makeStyles((theme) => ({
+  icons: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
+const WorkDoneCard = ({ title, changelogURL }: WorkDoneCardProps) => {
+  const [markdownString, setMarkDownString] = React.useState<string>('');
```

Comment:
Importer `useState` fra React istedenfor å bruke `React.useState`. Du trenger ikke deklarere at det er en string type i useState når den initieres med `''`.

Dette bør altså være sånn:
```suggestion
  const [markdownString, setMarkDownString] = useState('');
```